### ISSUE TITLE
print_num(x) function should cast its argument to an int

### ISSUE THREAD

**Issue by cmwedin** [inkle/ink-library#17]
This is a very minor issue and very easy to change on your own, so i understand if it isn't a priority, but it would make sense to want to print out a float in text as the closest integer. As it stands float arguments will be accepted but not print properly for inputs higher than 20 as integer division isn't used in the conditional.

**Comment by joningold** [comment]
Good idea! We're actually rolling this function into inky's "ink snippets" menu in the next version, so I'll include this tweak there. (This repo is becoming somewhat redundant now ink has an internal library.)
